### PR TITLE
docs(website): Document session lifetimes

### PR DESCRIPTION
**Pull request by jamilbk**
Have had a few customers ask about this now.

Ideally all lifetimes would be configurable. See #3179

### CODE REVIEW

File: website/src/app/kb/authenticate/readme.mdx [hunk 1]
```diff
@@ -5,3 +5,17 @@ Firezone supports the following authentication methods:
 1. [Email (OTP)](/kb/authenticate/email)
 1. [OpenID Connect (OIDC)](/kb/authenticate/oidc)
 1. [Google Workspace](/kb/authenticate/google)
+
+## Session lifetime
+
+The below session lifetimes apply to all identity providers:
+
+- Admin portal: **4 hours**
+- Client application: **7 days**
```
Comment (bmanifold) [inline website/src/app/kb/authenticate/readme.mdx]: I noticed you mentioned in the PR description that ideally all lifetimes would be configurable.  Is that something that's on the roadmap?  If so, would it make sense to add a small note saying these will be configurable in the future?

File: website/src/app/kb/authenticate/readme.mdx [hunk 2]
```diff
@@ -5,3 +5,17 @@ Firezone supports the following authentication methods:
 1. [Email (OTP)](/kb/authenticate/email)
 1. [OpenID Connect (OIDC)](/kb/authenticate/oidc)
 1. [Google Workspace](/kb/authenticate/google)
+
+## Session lifetime
+
+The below session lifetimes apply to all identity providers:
+
+- Admin portal: **4 hours**
```
Comment (AndrewDryga) [inline website/src/app/kb/authenticate/readme.mdx]:
```suggestion
- Admin portal: IdP access token lifetime but up to **10 hours**
```

**Review by bmanifold — APPROVED** [review]
LGTM.  The one question I had is not blocking.

File: website/src/app/kb/authenticate/readme.mdx [hunk 3]
```diff
@@ -5,3 +5,17 @@ Firezone supports the following authentication methods:
...
+- Client application: **7 days**
+- Service account: **365 days** (by default, configurable)
```
Comment (AndrewDryga) [inline website/src/app/kb/authenticate/readme.mdx]:
```suggestion
- Service account: **365 days** (by default, configurable per token)
```

**Comment by AndrewDryga** [comment]
@jamilbk removed from merge queue because the durations are not accurate

File: website/src/app/kb/authenticate/readme.mdx (hunk 1 above)
Comment (jamilbk) [reply website/src/app/kb/authenticate/readme.mdx]: It's not specifically listed on the public roadmap, but there's an issue for it here: #3179 

> would it make sense to add a small note saying these will be configurable in the future?

Hmm, as a general approach I've been trying to keep the future state out of the docs unless we want to solicit input on it. For this one I think it's mostly well-defined, just need to prioritize it

File: website/src/app/kb/authenticate/readme.mdx (hunk 2 above)
Comment (jamilbk) [reply website/src/app/kb/authenticate/readme.mdx]: 👍

**Pull request merged**